ambari/README.md

```diff
@@ -0,0 +1,61 @@
+# Agent Check: Ambari
+
+## Overview
+
+This check monitors [Ambari][1] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The Ambari check is included in the [Datadog Agent][7] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `ambari.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your ambari performance data. See the [sample ambari.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3].
+
+### Validation
+
+[Run the Agent's status subcommand][4] and look for `ambari` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+If host metrics collection is enabled with `collect_host_metrics` this integration will collect
+for every host in every cluster the following system metrics:
+* boottime
+* cpu
+* disk
+* memory
+* load
+* network
+* process
+
+If service metrics collection is enabled with `collect_service_metrics` this integration will collect for each
+whitelisted service component the metrics with headers in the white list.
+
```

Comment:
This text has to be above the `### Metrics` header. Then add this to the metrics section:

```suggestion
See [metadata.csv][8] for a list of metrics provided by this integration.
```